web/src/engine/predictive-text/worker-thread/src/tests/mocha/cases/edit-distance/context-state.js

```diff
@@ -0,0 +1,53 @@
+import { assert } from 'chai';
+
+import { ContextState } from '#./correction/context-state.js';
+import * as models from '#./models/index.js';
+
+import { default as defaultBreaker } from '@keymanapp/models-wordbreakers';
+
+import { jsonFixture } from '@keymanapp/common-test-resources/model-helpers.mjs';
+
+var TrieModel = models.TrieModel;
+
+var plainModel = new TrieModel(jsonFixture('models/tries/english-1000'),
+  {wordBreaker: defaultBreaker});
+
+describe('ContextState', () => {
+  it('<constructor>', () => {
+    let context = { left: '', right: '' };
+    let state = new ContextState(context, plainModel);
+
+    assert.equal(state.context, context);
+    assert.equal(state.model, plainModel);
+    assert.isOk(state.tokenization);
+    assert.isUndefined(state.isManuallyApplied);
+    assert.isNotOk(state.suggestions);
+    assert.isNotOk(state.appliedSuggestionId);
+  });
+
+  describe('initializing without prior tokenization', () => {
+    it('<empty context>', () => {
```

Comment:
test names should describe expected outcome, e.g. `it('should have 1 token with an empty context')` or something like that. Here and below.